### ISSUE TITLE
[Enhancement] Visually distinguish warm-up sets in workout summary

### ISSUE THREAD

**Issue by JonathanGuberman** [cfilipov/MuscleBook#33]
It'd be really nice to tell at a glance which sets were warm-up and which were working sets in the summary.

**Comment by cfilipov** [comment]
Yes. In addition to this I also wanted to make it easy to see activation on each set. The idea I had floating around in my head: on the left side of each workset row on the workout summary there is about 15 points of empty space, I was thinking of having a simple colored region similar to [this screenshot](https://github.com/insidegui/WWDC/blob/master/screenshots/screenshot-schedule-fake.png) (except without the left, top and bottom padding). Yellow or orange would indicate warmup, and various shades of red would correspond to that set's activation level. This would let you quickly see not only if the set was a warmup, but also if the set was light, intense or a PR (full red).

**Comment by cfilipov** [comment]
One more thing. I had this in a very early version of the app (before open sourcing) but got rid of it temporarily:

I was thinking of having rows between each set to indicate rest. This way you can see individual rest times between each set. This could get visually distracting especially if you do a lot of sets, so there would also have to be a toggle to show/hide rest rows (I had this too). 

Alternately: I plan to eventually have a graph on both the workout summary and the daily summary that visually displays the same information, here's an ASCII art representation of what the row will look like:

```
+-------------------------------------------------------+
| ||    |||||    |||   ||           |||     |   ||  |   |
+-------------------------------------------------------+
```

Each tick would be either yellow/orange (warmup) or some shade of red (again, corresponding to activation). The space between ticks is rest time, the thicker the tick the longer the duration of the set. This gives you all the same data as having the rest rows, but it's easier to see, but you don't get the details (like how many actual min of rest were between set 1 and set 2?).

**Comment by JonathanGuberman** [comment]
I'm very enthusiastic about a graph like that; it was actually one of the things that attracted me to this app, because I could export the data to make the graph myself. I was thinking something similar, but with small bars to indicate the weight (probably as percentage of 1RM to avoid scale issues from different exercises, and to obviate the need for scale indicators). Something like this:

```
+---------------------------------------------------------------------------------+
|              GG   GG   GGG                   BB  BBB               RRR    RRRR  |
|     gg ggg   GG   GG   GGG              bbb  BB  BBB         rrrr  RRR    RRRR  |
| gg  gg ggg   GG   GG   GGG           b  bbb  BB  BBB         rrrr  RRR    RRRR  |
| gg  gg ggg   GG   GG   GGG    bb     b  bbb  BB  BBB   rrr   rrrr  RRR    RRRR  |
+---------------------------------------------------------------------------------+
```

where the different letters represent different colours for different exercises, and the lower-case vs upper-case indicates different shades or fills to indicate warm-up vs work sets. As in yours, spacing is for timing, and as in yours it would remove numbers so as to give a brief visual overview of the workout rather than the exact numbers.
